media/js/src/main.jsx

```diff
@@ -88,9 +88,11 @@ class Main extends React.Component {
     render() {
         return (
             <div>
-                <nav className="nav nav-pills nav-justified" role="tablist">
+                <nav id="three-section-tabs"
```

Comment:
This needs a little padding at the bottom. Maybe at a margin-bottom: 20px or so?